inspirehep/modules/workflows/tasks/matching.py

```diff
@@ -87,9 +88,12 @@ def fuzzy_match(obj, eng):
     """
     if not current_app.config.get('FEATURE_FLAG_ENABLE_FUZZY_MATCHER'):
         return False
-
+    math_ml_latex_regex = '<mml[^>]*>([^<]+)<\/mml> | <latex[^>]*>([^<]+)<\/latex>'
```

Comment:
Both sides of these regexes are wrong.
* LaTeX is not an XML language, so isn't delimited by XML-like tags. Instead, it uses `$...$` or `\(...\)` or `\[...\]` as delimiters. You should use some variation on https://github.com/inspirehep/inspirehep/blob/cb15e67376042277a0550a2ae0f8b7147a8d65ae/backend/inspirehep/records/marshmallow/literature/utils.py#L20 (also adding the last pattern I pointed out).
* where is your examples coming from? `mml` is an XML namespace prefix, not the actual tag. In INSPIRE (and [HTML5](https://github.com/inspirehep/inspirehep/blob/cb15e67376042277a0550a2ae0f8b7147a8d65ae/backend/inspirehep/records/marshmallow/literature/utils.py#L20)), the namespace shouldn't be included. You should match `<math>` tags instead (and use a more realistic example in the tests, like the abstract of https://inspirehep.net/api/literature/1870659). Also your regex should be non-greedy (currently you're matching everything in `<mml>1</mml>middle text outside<mml>2</mml>`, but the middle text outside of the tags shouldn't be). It's also not clear to me if you want to strip all markup, or strip also the content of the tags (with my previous example, do you expect `1middle text outside2` or `middle text outside`).